### PR TITLE
Added recipe for pyramid_debugtoolbar and pyramid_mako

### PR DESCRIPTION
**Pull request by ChrisBarker-NOAA**
@conda-forge/help-python: this is because defaults hasn't updated since py3.8

And I've made the a noarch package now.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

@JamesMakela-NOAA: please post a comment indicating willingness to maintain.

### CODE REVIEW

File: recipes/pyramid_mako/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - pyramid_mako
+  # requires:
+  #   - webtest >=1.3.1
+
+about:
+  home: https://trypyramid.com
+  license: BSD-derived
```
Comment (chrisburr) [inline recipes/pyramid_mako/meta.yaml:35]: From the bot: 

>  * License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.
> 
> Documentation on acceptable licenses can be found [here]( https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions ).

File: recipes/pyramid_mako/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: The Pyramid Web Framework, a Pylons project
+  doc_url: https://docs.pylonsproject.org/projects/pyramid_mako/en/latest/
+  dev_url: https://github.com/Pylons/pyramid_mako
+
+extra:
+  recipe-maintainers:
+    - ChrisBarker-NOAA
+    - JamesMakela-NOAA
```
Comment (chrisburr) [inline recipes/pyramid_mako/meta.yaml:45]: @JamesMakela-NOAA Are you happy to be listed as a recipe maintainer here?

File: recipes/pyramid_mako/meta.yaml (hunk 1 above)
Comment (ChrisBarker-NOAA) [reply recipes/pyramid_mako/meta.yaml:35]: This is a question I'm unclear about with conda-forge policy:

What do we do if the license of the software has no SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

In this case, it's the "Pylons Licence" which, IIUC, is pretty much BSD, but not exactly.

So how DO we put that in a conda-forge recipe?

Comment (chrisburr) [reply recipes/pyramid_mako/meta.yaml:35]: It's always possible to have a LicenseRef as the remainder of the string is then arbitrary e.g. `LicenseRef-Pylons`

File: recipes/pyramid_mako/meta.yaml (hunk 2 above)
Comment (JamesMakela-NOAA) [reply recipes/pyramid_mako/meta.yaml:43]: > @JamesMakela-NOAA Are you happy to be listed as a recipe maintainer here?

Yes, that would be fine.

**Comment by ChrisBarker-NOAA** [comment]
Whoo hoo! finally all good.

**Review by JamesMakela-NOAA — APPROVED** [review]
Ok, I viewed the files and they look pretty good to me.

**Pull request merged**